administration/src/components/BasicDialog.tsx

```diff
@@ -0,0 +1,45 @@
+import { Icon, IconSize } from '@blueprintjs/core'
```

Comment:
Only use @mui for the application form

Reply:
gonna be reverted since we use end screen instead of dialog